public/templates/data.tmpl

```diff
@@ -88,8 +92,13 @@
           "type": "raster-dem",
           "url": "{{public_url}}data/{{id}}.json",
           "encoding": "{{terrain_encoding}}"
+        },
+        "contour": {
+          "type": "vector",
+          "tiles": [ baseUrl + "/data/{{id}}/contour/{z}/{x}/{y}" ],
```

Comment:
One thing i would like to do is make a json endpoint for the contour lines, so this does not need baseurl